infra/taskcluster-hook-pipeline-start.json

```diff
@@ -29,13 +29,15 @@
             "cache": {},
             "capabilities": {},
             "command": [
-                "/hello"
+                "/bin/bash",
+                "-lcx",
+                "git clone https://github.com/mozilla/bugbug && cd bugbug && git checkout master && pip install taskcluster json-e requests pyyaml && python infra/spawn_data_pipeline.py data-pipeline.yml"
```

Comment:
We should have an image for the hook task, so that we don't have to clone and pip install whenever the hook is triggered.